pkg/reconciler/trigger/resources/binding.go

```diff
@@ -88,14 +85,14 @@ func NewBinding(ctx context.Context, args *BindingArgs) (*rabbitv1beta1.Binding,
 // BindingLabels generates the labels present on the Queue linking the Broker / Trigger to the
 // Binding.
 func BindingLabels(b *eventingv1.Broker, t *eventingv1.Trigger) map[string]string {
-	if t == nil {
```

Comment:
Why these changes?

Reply:
`ExchangeLabels`, `QueueLabels` and `BindingLabels` are identical functions.

`ExchangeLabels` had one implementation, while `QueueLabels` and `BindingLabels` had another. This was my first step to unify them, as a precursor to maybe extracting into a single place.

I can revert if you think doing nothing is better than this change.

Reply:
Let's do this in a separate PR